.github/lighthouse/lighthouserc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ci": {
+        "collect": {
+            "settings": {
+                "onlyCategories": ["performance"],
```

Comment:
```suggestion
                "onlyCategories": [
                    // Excluded categories do not currently pass
                    // "accessibility",
                    // "best-practices",
                    "performance",
                     // "pwa",
                     // "seo"
                ],
```
A

Reply:
I don't think we can add comments in json. :(

Reply:
Can we use a JS config like this one? https://github.com/GoogleChrome/lighthouse/blob/main/docs/configuration.md#usage

Reply:
Updated to
```
export default {
    extends: 'lighthouse:default',
    settings: {
        onlyCategories: [
            // Excluded categories do not currently pass
            // "accessibility",
            // "best-practices",
            'performance',
            // "pwa",
            // "seo"
        ],
        onlyAudits: ['first-meaningful-paint', 'speed-index', 'interactive'],
    },
};
```